src/store/flowContext.ts

```diff
@@ -98,66 +103,116 @@ export const updateContactFields = (
     }
 });
 
+// tslint:disable-next-line:no-shadowed-variable
 export const updateGroups = (groups: SearchResult[]): UpdateGroupsAction => ({
     type: Constants.UPDATE_GROUPS,
     payload: {
         groups
     }
 });
 
+// tslint:disable-next-line:no-shadowed-variable
 export const updateResultNames = (resultNames: CompletionOption[]): UpdateResultNamesAction => ({
     type: Constants.UPDATE_RESULT_NAMES,
     payload: {
         resultNames
     }
 });
 
+// tslint:disable-next-line:no-shadowed-variable
 export const updateComponents = (components: Components): UpdateComponentsAction => ({
     type: Constants.UPDATE_COMPONENTS,
     payload: {
         components
     }
 });
 
-// Reducer
-export default (state: FlowContext = initialState, action: ActionTypes) => {
+// Reducers
+export const definition = (
```

Comment:
breaking these into individual reducers, which feels better to me from an OCD standpoint but i also observed a coverage increase compared to the single-reducer approach 🤷‍♂️